### PR TITLE
Feature/weng-226-setup-grid

### PR DESCRIPTION
**Pull request by itsamoreh**
### DESCRIPTION

- This PR will add grid classes to the tailwind imports file using `@layer`.
- This PR will also add containers to templates where they are needed to constrain content to container width

### SCREENSHOT

screen-capture 2022-10-05 at 2 52 33 PM

### STEPS TO VERIFY

Verify that content like the header and footers are inside a container.

### DOCUMENTATION

Updated on wiki branch `feature/wiki-amor`

### CODE REVIEW

File: template-parts/content.php [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 <article <?php post_class( 'post-container' ); ?>>
 
-	<header class="entry-header">
+	<header class="entry-header container">
```
Comment (webdevstudios-hosting) [inline template-parts/content.php]:
```suggestion
	<header class="entry-header">
```

Container is already applied.

File: template-parts/content.php [hunk 2]
```diff
@@ -58,7 +58,7 @@
 		?>
 	</div><!-- .entry-content -->
 
-	<footer class="entry-footer">
+	<footer class="entry-footer container">
```
Comment (webdevstudios-hosting) [inline template-parts/content.php]:
```suggestion
	<footer class="entry-footer">
```

Container is already applied

**Review by webdevstudios-hosting — CHANGES_REQUESTED** [review]
Just a couple suggestions.

**Review by oliverharrison — APPROVED** [review]
This looks good to me.

**Review by oliverharrison — APPROVED** [review]

**Pull request merged**